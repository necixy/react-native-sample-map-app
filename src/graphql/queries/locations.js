import gql from "graphql-tag";
export default gql`
  query {
    locations {
      id
      name
      address
      lat
      lng
    }
  }
`;
