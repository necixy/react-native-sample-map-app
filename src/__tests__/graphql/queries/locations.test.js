import locations from "../../../graphql/queries/locations";

describe("Locations Query Testing", () => {
  it("Locations Query Snapshots correctly", () => {
    expect(locations).toMatchSnapshot();
  });
});
