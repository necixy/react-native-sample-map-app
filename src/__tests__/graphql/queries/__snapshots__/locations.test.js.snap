// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Locations Query Testing Locations Query Snapshots correctly 1`] = `
Object {
  "definitions": Array [
    Object {
      "directives": Array [],
      "kind": "OperationDefinition",
      "name": undefined,
      "operation": "query",
      "selectionSet": Object {
        "kind": "SelectionSet",
        "selections": Array [
          Object {
            "alias": undefined,
            "arguments": Array [],
            "directives": Array [],
            "kind": "Field",
            "name": Object {
              "kind": "Name",
              "value": "locations",
            },
            "selectionSet": Object {
              "kind": "SelectionSet",
              "selections": Array [
                Object {
                  "alias": undefined,
                  "arguments": Array [],
                  "directives": Array [],
                  "kind": "Field",
                  "name": Object {
                    "kind": "Name",
                    "value": "id",
                  },
                  "selectionSet": undefined,
                },
                Object {
                  "alias": undefined,
                  "arguments": Array [],
                  "directives": Array [],
                  "kind": "Field",
                  "name": Object {
                    "kind": "Name",
                    "value": "name",
                  },
                  "selectionSet": undefined,
                },
                Object {
                  "alias": undefined,
                  "arguments": Array [],
                  "directives": Array [],
                  "kind": "Field",
                  "name": Object {
                    "kind": "Name",
                    "value": "address",
                  },
                  "selectionSet": undefined,
                },
                Object {
                  "alias": undefined,
                  "arguments": Array [],
                  "directives": Array [],
                  "kind": "Field",
                  "name": Object {
                    "kind": "Name",
                    "value": "lat",
                  },
                  "selectionSet": undefined,
                },
                Object {
                  "alias": undefined,
                  "arguments": Array [],
                  "directives": Array [],
                  "kind": "Field",
                  "name": Object {
                    "kind": "Name",
                    "value": "lng",
                  },
                  "selectionSet": undefined,
                },
              ],
            },
          },
        ],
      },
      "variableDefinitions": Array [],
    },
  ],
  "kind": "Document",
  "loc": Object {
    "end": 91,
    "start": 0,
  },
}
`;
